docs/installation/server-config/config-reference.rst

```diff
@@ -147,6 +147,16 @@ ksql.sink.replicas
 
 The default number of replicas for the topics created by KSQL. The default is one.
 
+------------------------------------
+ksql.functions.substring.legacy.args
+------------------------------------
+
+Controls the semantics of the SUBSTRING UDF. Refer to the SUBSTRING documentation in the :ref:`function <functions>` guide for details.
+
+When upgrading headless mode KSQL applications from versions 5.0.0 or earlier without updating your queries that use SUBSTRING to match 
```

Comment:
Should 5.0.0 be 5.0.1 now, given that 5.0.1 was released?

Reply:
I'll update to 5.0.x